### ISSUE TITLE
Must-gather helper pod remains stuck during teardown

### ISSUE THREAD

**Issue by am-agrawa** [red-hat-storage/ocs-ci#7130]
[test_noobaa_postgres_cm_post_ocs_upgrade](https://reportportal-ocs4.apps.ocp-c1.prod.psi.redhat.com/ui/#OCS/launches/427/8811/391098/391105/391107/log) --- Run ID: 1675262490 || Suite: tier1-or-tier_after_upgrade || ODF 4.10.10-2, OCP 4.10, aws

[test_noobaa_postgres_cm_post_ocs_upgrade](https://reportportal-ocs4.apps.ocp-c1.prod.psi.redhat.com/ui/#OCS/launches/427/8810/391052/391091/391093/log) --- Run ID: 1675241829 || Suite: pre_upgrade-or-pre_ocs_upgrade-or-ocs_upgrade-or-post_ocs_upgrade-or-post_upgrade || ODF 4.10.9-7, OCP 4.10, aws

Test passes but fails during teardown

**Comment by jilju** [comment]
is this the issue ?
https://github.com/red-hat-storage/ocs-ci/issues/5618

**Comment by am-agrawa** [comment]
> is this the issue ? #5618

yes, looks similar

**Comment by ebenahar** [comment]
@OdedViner is this fixed by one of your recent changes?

**Comment by OdedViner** [comment]
@ebenahar No, I wanted to delete must-gather ns... but we are running mg in parallel in DR... https://github.com/red-hat-storage/ocs-ci/pull/7314

**Comment by am-agrawa** [comment]
Also hit on `Run ID: 1682673833 || Suite: pre_upgrade-or-pre_ocp_upgrade-or-pre_ocs_upgrade-or-ocp_upgrade-or-ocs_upgrade-or-post_ocp_upgrade-or-post_ocs_upgrade- || ODF 4.12.2-4, OCP 4.12, aws`

**Comment by petr-balogh** [comment]
As we discussed it in chat, and I suggested to ignore those pods as well, now I see it's been excluded here:
https://github.com/red-hat-storage/ocs-ci/pull/7539

If that's fixing the issue , I think we can close it here.

**Comment by ebenahar** [comment]
@AviadP is this still relevant? Please update

**Comment by AviadP** [comment]
it been a while since i sew this issue, closing it for now
